haskell/src/Monocle/Api/Server.hs

```diff
@@ -299,6 +299,32 @@ crawlerCommitInfo request = do
         . Enumerated
         $ Right err
 
+-- | /suggestions endpoint
+searchSuggestions :: SearchPB.SuggestionsRequest -> AppM SearchPB.SuggestionsResponse
+searchSuggestions request = do
+  tenants <- getConfig
+  let SearchPB.SuggestionsRequest {..} = request
+
+  let tenantM = Config.lookupTenant tenants (toStrict suggestionsRequestIndex)
+
+  case tenantM of
+    Just tenant -> do
+      now <- getCurrentTime
+      suggestions <- runTenantQueryM tenant (emptyQ now) $ do Q.getSuggestions
```

Comment:
nit: no need for `$ do`:
```suggestion
      suggestions <- runTenantQueryM tenant (emptyQ now) Q.getSuggestions
```

Reply:
Actually it should be `suggestions <- runTenantQueryM tenant (emptyQ now) $ Q.getSuggestions tenant`